README.md

```diff
@@ -1,2 +1,178 @@
+3. Copy a value of a cookie with the name: "service-jwt-0". It is your's teacher account access token.
+
+## JSON requests
+
+JSON requests are the easiest point to start. However, we are recommending to use binary requests.  
+Note: **POST is the only supported method**
+
+**Public AccountService.login call**:
+```
+curl -v https://www.classtime.com/service/public/Account/login \
+    -X POST \
+    -H 'Content-Type: application/json' \
+    -H 'Accept: application/json,*/*' \
+    -d '{ provider: "Google", role: "Teacher" }'
+```
+```
+Response: { "redirectUrl": "https://accounts.google.com..." }
+```
+
+**Restricted AccountService.getPublicUserProfile call**:
+Pre-requirement: you must know your account id and access token. You could obtain it following [this small guide](https://github.com/gopollock/public-api/blob/main/flow-examples/my-account-id.md).
+
+```
+curl -v https://www.classtime.com/service/public/Account/getPublicUserProfile \
+    -X POST \
+    -H 'Content-Type: application/json' \
+    -H 'Accept: application/json,*/*' \
+    -H 'Authorization: JWT ${yourTeacherAccessToken}' \
+    -d '{ account_id: ${yourAccountId} }'
+```
+```
+Response: {
+    "email": "you@email.com",
+    userProfile: {
+      "firstName": "John",
+      "lastName": "White",
+      "picture": "",
+    }
+    ...,
+}
+```
+
+## Binary requests
+
+Pre-requirement for binary requests are generated protobuf. They would help you to serialize requests and deserialize responses.  
+The full process of protobuf generation and usage is described in [generating protos guide](https://github.com/gopollock/public-api/blob/main/generate-protos.md).
+
+**Public AccountService.login call**:
+```
+curl -v https://www.classtime.com/service/Account/login \
+    -X POST \
+    -H 'Content-Type: application/protobuf' \
+    -H 'Accept: application/protobuf,*/*' \
+    -d '' \
+    -o output.bin | tee output.bin
+```
+Response: `LoginResponse` in binary format
+
+`-d ''` is an empty binary. It will result in default values for all fields.  
+You could replace it with your serialized value.
+
+**Restricted AccountService.getPublicUserProfile call**:
+Pre-requirement: you must know your account id and access token. You could obtain it following [this small guide](https://github.com/gopollock/public-api/blob/main/flow-examples/my-account-id.md).
+
+```
+curl -v https://www.classtime.com/service/public/Account/getPublicUserProfile \
+    -X POST \
+    -H 'Content-Type: application/protobuf' \
+    -H 'Accept: application/protobuf,*/*' \
+    -H 'Authorization: JWT ${yourTeacherAccessToken}' \
+    -d ${protobufBinaryRequestData} \
+    -o output.bin | tee output.bin
+
+```
+Response: `UserProfileResponse` in binary format
+
+## Rate limits
+
+Classtime limits requests to 800 (requests / min) per IP address or authorized account.
```

Comment:
Better be more explicit: It is either
- per IP address for unauthorized requests or
- per user for authorized requests.